src/actions/wordpress-dotorg-plugin-deploy.md

```diff
@@ -1,7 +1,7 @@
 ---
 path: "/wordpress-dotorg-plugin-deploy"
 title: "WordPress.org Plugin Deploy"
-github_url: "https://github.com/10up/actions-wordpress/dotorg-plugin-deploy"
+github_url: "https://github.com/10up/actions-wordpress/tree/develop/dotorg-plugin-deploy"
```

Comment:
```suggestion
github_url: "https://github.com/10up/actions-wordpress"
```
We might need to move this to this url, for the stars to work, did you want to try that?